dds_cli/project_status.py

```diff
@@ -148,6 +148,112 @@ def update_status(self, new_status, deadline=None, is_aborted=False, no_mail=Fal
+        print_info = (
+            f"\nCurrent deadline: [b][green]{current_deadline}[/green][/b]\n"
+            f"Default deadline extension: [b][green]{default_unit_days}[/green][/b] days\n"
+        )
+        table = self.generate_project_table(project_info=response_json.get("project_info"))
+        dds_cli.utils.console.print(table)
+        dds_cli.utils.console.print(print_info)
+
+        # First question, number of days to extend the deadline
+        prompt_question = (
+            f"Enter the number of days you want to extend the project, "
+            f"the number of days has to be equal or same as "
+            f"[b][green]{default_unit_days}[/green][/b].\n"
+            f"Or leave it empty to apply the default "
+            f"[b][green]{default_unit_days} days [/green][/b]"
+        )
+
+        dds_cli.utils.console.print(prompt_question)
+        extend_deadline = rich.prompt.Prompt.ask("-")
+        if not extend_deadline:
+            # Set extend_deadline to default
+            extend_deadline = default_unit_days
+        try:
+            # the input was an string --> convert to integer
+            extend_deadline = int(extend_deadline)
+            if extend_deadline > default_unit_days:
+                dds_cli.utils.console.print(
+                    "\n[b][red]The number of days has to be lower than "
+                    "the default deadline extension number[/b][/red]\n"
+                )
+                LOG.info("Exiting the function, try again")
+                sys.exit(0)
+
+        except ValueError:
+            dds_cli.utils.console.print(
+                "\n[b][red]Remember to write the number of days "
+                "using numbers (dont use letters)[/b][/red]\n"
+            )
+            LOG.info("Exiting the function, try again")
+            sys.exit(0)
+
+        # Second question, confirm operation
+        prompt_question = (
+            f"\n\n[b][blue]Are you sure [/b][/blue]you want to perform this operation?. "
+            f"\nThis will extend the deadline by [b][blue]{extend_deadline} days[/b][/blue]."
+            "\nYou can only extend the data availability a maximum of "
+            "[b][blue]3 times[/b][/blue], this consumes one of those times."
+        )
+
+        dds_cli.utils.console.print(prompt_question)
+        if not rich.prompt.Confirm.ask("-"):
+            LOG.info("Probably for the best. Exiting.")
+            sys.exit(0)
+
+        # Update parameters for the second request
+        extra_params = {**extra_params, "confirmed": True, "new_deadline_in": extend_deadline}
+
+        response_json, _ = dds_cli.utils.perform_request(
+            endpoint=DDSEndpoint.UPDATE_PROJ_STATUS,
+            headers=self.token,
+            method="patch",
+            params={"project": self.project},
+            json=extra_params,
+        )
+        dds_cli.utils.console.print(f"Project {response_json.get('message')}")
```

Comment:
* Check that the message has actually been returned (maybe something else was, maybe we've changed something which hasn't been caught in the `perform_request` for example)
* Use the logging functionality instead of print 
```suggestion
        message = response_json.get("message")
        if not message:
            raise DDSCLIException("No message returned from API. Cannot verify extension of project deadline.")
            
        LOG.info(message)
```